topics/releases.md

```diff
@@ -1,144 +1,98 @@
+
+The Valkey community strives to make as few backwards breaking changes as possible.
+When breaking changes are required, we will also strive to provide a way to mitigate the impact without incuring downtime to your application.
 
 ## Release schedule
 
-A new major version is planned for release once a year.
+The Valkey community strives to release a stable major version once a year.
+Stable minor versions are created as needed in between major releases, and we aim to release at least one minor version a year.
 
-Generally, every major release is followed by a minor version after six months.
+### Release candidate
 
-Patches are released as needed to fix high-urgency issues, or once a stable
-version accumulates enough fixes to justify it.
+New minor and major versions of Valkey begin by branching off the `unstable` branch as an initial release candidate branch, which take the form *major.minor.patch-R#*.
+The first release candidate, or RC1, is released once it can be used for development purposes and for testing the new version.
+At this stage, most of the new features and changes in the new version are ready for review, and the version is released for the purpose of collecting the public's feedback.
+Subsequent release candidates are released every coupe of weeks, primarily for fixing bugs and refining features based off of user input.
 
-For contacting the core team on sensitive matters and security issues, please
-see [SECURITY.md](https://github.com/valkey-io/valkey/blob/unstable/SECURITY.md).
+### Stable release
 
-## Support
+Once development has ended and the feedback for release candidate slows down, it is ready for the final release. 
+At this point, the release is marked as stable and is released with "0" as its patch-level version.
 
-As a rule, older versions are not supported as we try very hard to make the
-API mostly backward compatible.
+Patches are released as needed to fix high-urgency issues, or once a stable version accumulates enough fixes to justify it.
 
-Upgrading to newer versions is the recommended approach and is usually trivial.
+## Support
 
 The latest stable release is always fully supported and maintained.
 
-Two additional versions receive maintenance only, meaning that only fixes for
-critical bugs and major security issues are committed and released as patches:
+The Valkey community will provide maintanence support, providing patch releases for bug fixes and all security fixes, for 3 years from when a version was first released.
+
+The Valkey community will also provide extended security security support for the latest minor version of each major version for 5 years from when a version was first released.
+The Valkey community will only backport security issues we believe to be possible to exploit, which will be up to the discretion of the TSC.
+
+For contacting the core team on sensitive matters and security issues, please see [SECURITY.md](https://github.com/valkey-io/valkey/blob/unstable/SECURITY.md).
```

Comment:
Here, "TCS" and "core team" are used as synonyms, but I don't know if all readers know that.